packages/api-server/src/methods/validator.ts

```diff
@@ -427,17 +423,6 @@ export function verifyEstimateGasCallObject(
   return undefined;
 }
 
-export function verifyStorageKey(
-  key: string,
-  index: number
-): InvalidParamsError | undefined {
-  const err = verifyHexString(key, index);
```

Comment:
why not change `verifyHexString` to `verifyHexNumber` here? it is more simple and `padContext("storageKey")` won't be lost?

Reply:
ah I see, it is more of a position number than a storage key concept in godwoken